judo-runtime-core-dao-rdbms/src/main/java/hu/blackbelt/judo/runtime/core/dao/rdbms/executors/SelectStatementExecutor.java

```diff
@@ -380,6 +380,9 @@ private void applyQueryCustomizer(final SubSelect query, final DAO.QueryCustomiz
         }
         if (queryCustomizer != null && queryCustomizer.getSeek() != null) {
             query.setLimit(queryCustomizer.getSeek().getLimit());
+            if (queryCustomizer.getSeek().getOffset() > 0) {
```

Comment:
If `0` is "default"

Reply:
If 0 is set nothing to do. -1 is to be sure it is not set.